### PR TITLE
SQL views guidance 

### PR DESCRIPTION
**Pull request by alexsnowdon**
Merge request template: please remove the appropriate parts of this template. 

Pre-merge request checklist (to be completed by the one making the request):
 - [x] I have performed a full review of this code myself.
    - For Python code in PySpark specific sections, all code should have been run in Jupyter notebooks. 
    - For code in sections of the book containing both Python and R code, the page of the book should be constructed as described in the [contributing guide](https://github.com/best-practice-and-impact/ons-spark/blob/main/CONTRIBUTING.md) and converted to a markdown file. 
 - [x] editing the code was not required here but I did check the code fits with the formatting in the rest of the book
 - [x] I have built the book as outlined in the [contributing guide](https://github.com/best-practice-and-impact/ons-spark/blob/main/CONTRIBUTING.md) and confirmed that any additional/modified content is displaying as expected. Unable to build due to blockers on computer. 

Details of this request (such as):
 - I have added a new page into the Ancillary section of the book called 'Creating SQL views in HDFS'
 - I have added two new images to the 'images' folder that are referenced in the new page
 - The page is based on the guidance on our Gitlab under how-to/SQL views. I have edited the wording and added new sections to give more information on SQL views. 

Requirements for review (such as):
 - Read the content to make sure you it makes sense and covers the important bits. 
 - Check the code runs (none of the code has been edited). 
 - Make sure the images show correctly (x2).
 - Please build the book and make sure the .html shows correctly (I have been unable to test this).

### CODE REVIEW

**Review by Jday7879 — CHANGES_REQUESTED** [review]
Book builds and images are displayed correctly.  Other than a small handful of grammatical errors in the first section and other suggestions this is good to merge.
Really nice work Alex

File: ons-spark/ancillary-topics/creating-a-SQL-view-in-HDFS.ipynb [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    "# Creating a SQL view in HDFS"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A view in SQL is a way of storing a query by creating a virtual table. This can be useful for many reasons, one being that you can more easily get your desired output as it is simpler to access a view than typing a query from scratch repeatedly. For more information on SQL views please refer to [Views in SQL](https://learnsql.com/blog/sql-view/).\n",
+    "\n",
+    "As mentioned above, SQL views can be used to avoid the re-running or pre-processing everytime you would like to use the same data (that has been created using a query) and avoids saving an entirely new table. An SQL view is simply a stoerd query that is applied to an underlying Hive table; **only** the metadata of the view will be stored and this happens after the query has been ran. Another advanateg of a is that it refreshes in real-time, which means if the underlying table updates then your view will, the reason being is that each time your view is called the statement used to create the view from the underlying table is re-ran. \n",
```
[the remark this answers is not in the document]
Comment (alexsnowdon) [reply ons-spark/ancillary-topics/creating-a-SQL-view-in-HDFS.ipynb]: Corrected typos as per feedback

File: ons-spark/ancillary-topics/creating-a-SQL-view-in-HDFS.ipynb [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    "    FROM rescue_clean\n",
+    "    GROUP BY animal_group\n",
+    "    ORDER BY average_cost DESC"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "![SQL_rescue_view_details](../images/SQL_rescue_view_details.PNG)\n",
```
[the remark this answers is not in the document]
Comment (alexsnowdon) [reply ons-spark/ancillary-topics/creating-a-SQL-view-in-HDFS.ipynb]: Added more details to the alternative text on figures as per the feedback

File: ons-spark/ancillary-topics/creating-a-SQL-view-in-HDFS.ipynb [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    "\n",
+    "In the screenshot above it can be seen that `rescue_view` has been saved as a `VIRTUAL_VIEW` table type as opposed to being a `MANAGED_TABLE` or `EXTERNAL_TABLE`. Another useful way to tell if you have a view is that in your filestore an 'eye-icon' will appear next to `rescue_view`. \n",
+    "\n",
+    "Your view, or virtual table, can now be called by using `SELECT * FROM rescue_view LIMIT 5` as seen in the screenshot below:\n",
+    "\n",
+    "![SQL_rescue_view_output](../images/SQL_rescue_view_output.PNG)\n",
```
[the remark this answers is not in the document]
Comment (alexsnowdon) [reply ons-spark/ancillary-topics/creating-a-SQL-view-in-HDFS.ipynb]: Added more details to the alternative text on figures as per the feedback

File: ons-spark/ancillary-topics/creating-a-SQL-view-in-HDFS.ipynb [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+   "metadata": {},
+   "source": [
+    "## Dropping a SQL view\n",
+    "\n",
+    "You may want to drop a view in SQL from the database, for example you no longer need it, you want to free up space or you want to over-write an existing view. To do the latter you need to drop the previously created view before creating another one with the same name. \n",
+    "\n",
+    "To drop an SQL view you simply write: `DROP VIEW view_name` so for us, in this example, it would be `DROP VIEW rescue_view`\n",
+    "\n",
+    "You can also use the following command: `DROP VIEW IF EXISTS view_name`. This code checked whether the view actually exists in your database and if so, it removes it. "
+   ]
```
[the remark this answers is not in the document]
Comment (alexsnowdon) [reply ons-spark/ancillary-topics/creating-a-SQL-view-in-HDFS.ipynb:130]: added further resources section to reflect the format of other book pages as per the feedback

**Review by Jday7879 — APPROVED** [review]
Accepting changes and merging into main

**Comment by Jday7879** [comment]
Book built correctly locally, but has not loaded images on the deployed version

**Pull request merged**